okapi-core/src/test/java/org/folio/okapi/ProxyTest.java

```diff
@@ -4537,6 +4537,35 @@ private void setupBasicModule(String tenant, String moduleId, String tenantPermi
         .then().statusCode(200).log().ifValidationFails();
   }
 
+  @Test
+  public void testObsoleteModules(TestContext context) {
```

Comment:
After the POST a GET /_/proxy/modules should be called to check that some obsolete module has been removed.